Backend/templates/gitlab-ci/master-gitlab-ci.yml

```diff
@@ -134,15 +154,20 @@ BuildAndUploadDockerDev:
 
 DeployChartDev:
   extends: .deploy_chart_template
+  environment:
+    name: development
   variables:
     PROJECT: $PROJECT_DEV
+    ENVIRONMENT: DEV
```

Comment:
Should the environment be in lower or upper case?

Reply:
i prefer upper case

Reply:
not sure if it was clear, but I was talking about the value of the variable 😅